src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -46,10 +57,12 @@ def create_configstore(client,
         public_network_access = 'Enabled' if enable_public_network else 'Disabled'
 
     if sku.lower() == 'free' and (enable_purge_protection or retention_days):
```

Comment:
Perhaps, add a check to confirm that any of the three new arguments have been specified?

Reply:
You are right. Updated